### ISSUE TITLE
[A11Y] Android: Collapsible HTML code tabs on components page are identified inappropriately when talkback reading granularity is set to “Headings and Landmarks”

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#389]
**ENVIRONMENT:**

OS (number): N (7.0)
Build number: NRD91R
Form factor (Smartphone / tablet):  Nexus 9
Browser: Chrome Dev 59.0.3062.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly Navigate to COMPONENTS link and double tap on it.
3. Swipe up or down with one finger to set the talkback reading granularity level to “Headings and Landmarks”
4. Try to swipe linearly forward on the page.

**ACTUAL RESULTS:**

With granularity set to “Headings and Landmarks”, HTML tab section is identified as region and user have no clue about this region.

**EXPECTED RESULTS:**

Collapsible HTML tabs should be identified appropriately when talkback reading granularity is set to “Headings and Landmarks”.

**ATTACHMENT(S):**

[Collapsible tabs.mp4.zip](https://github.com/ampproject/ampstart/files/914242/Collapsible.tabs.mp4.zip)
